### PR TITLE
Paginate Extended Query Tag Errors

### PR DESCRIPTION
**Pull request by wsugarman**
## Description
- Enable pagination for extended query tag errors with same `limit` and `offset` parameters as QIDO

## Related issues
[AB#85156](https://microsofthealth.visualstudio.com/Health/_workitems/edit/85156/)

## Testing
PR pipeline

### CODE REVIEW

File: src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.sql [hunk 1]
```diff
@@ -1960,6 +1967,9 @@ BEGIN
     INNER JOIN dbo.Instance AS I
     ON XQTE.Watermark = I.Watermark
     WHERE XQTE.TagKey = @tagKey
+    ORDER BY CreatedTime ASC
```
Comment (pengchen0692) [inline src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.sql:1970]: 2 concerns:
1. should we index `CreatedTime` if sort by?
2. What if there are multiple instances with same `CreatedTime`, would that return inconsistent result?

Comment (pengchen0692) [reply src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.sql:1970]: Looks like we should paginate on unique column https://stackoverflow.com/questions/57688411/sql-pagination-duplicates-with-ordering-by-non-unique-value, https://stackoverflow.com/questions/62133543/sql-server-paging-using-limit-and-offset-with-non-unique-ordering-column

Comment (pengchen0692) [reply src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.sql:1970]: Propose1: use `Watermark` instead, as it always unique for single tag
Propose2: still use `CreatedTime` but combination with `Watermark`, in this way, we still need to add index for them.

Comment (wsugarman) [reply src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.sql]: I added the index after all!

**Review by pengchen0692 — APPROVED** [review]

**Pull request merged**